js/ui/drop_down_button.js

```diff
@@ -103,15 +103,15 @@ let DropDownButton = Widget.inherit({
             onActionButtonClick: null,
 
             /**
-             * @name dxDropDownButtonOptions.onSelectionChanged
+             * @name dxDropDownButtonOptions.onSelectedItemChanged
```

Comment:
This name is confusing. You could think that something **in** the selected item has changed: a field was added or removed or a field value was updated. Because of this, and also because this option is called onSelectionChanged in other widgets, I suggest sticking to that name